helper/util.go

```diff
@@ -79,6 +80,37 @@ func EventByID(abiObject *abi.ABI, sigdata []byte) *abi.Event {
 	return nil
 }
 
+// AppendPubkeyPrefix returns publickey in uncompressed format
+func AppendPubkeyPrefix(signerPubKey []byte) []byte {
+	// append prefix - "0x04" as heimdall uses publickey in uncompressed format. Refer below link
+	// https://superuser.com/questions/1465455/what-is-the-size-of-public-key-for-ecdsa-spec256r1
+	prefix := make([]byte, 1)
```

Comment:
Should we ensure if the given input doesn't have the prefix or some other validation?

Reply:
That validation is out of this method. This method just adds a prefix when gets called.